packages/app/components/Queue/TA/TAQueueListDetail.tsx

```diff
@@ -102,6 +103,27 @@ export default function TAQueueListDetail({
   const selectedQuestion = allQuestionsList.find(
     (q) => q.id === selectedQuestionId
   );
+  const navigateQuestions = (isUp: boolean) => {
+    const priorityAndWaitingQuestionIds = [
+      ...questions.priorityQueue,
+      ...questions.queue,
+    ].map((question) => question.id);
+    const numOfQuestions = priorityAndWaitingQuestionIds.length;
+    if (numOfQuestions > 0) {
+      setSelectedQuestionId((prevId) => {
+        const addMinus = isUp ? -1 : 1;
+        const qIdIndex = priorityAndWaitingQuestionIds.indexOf(prevId);
+        const modulusMagic =
```

Comment:
LOL

Reply:
but also this is such a nice touch, it's super convenient to go to the bottom of the queue by pressing up at the top